src/Synapse/Log/LogServiceProvider.php

```diff
@@ -63,28 +96,20 @@ public function register(Application $app)
             $handlers[] = $this->rollbarHandler($app['environment']);
         }
 
-        $app['log'] = $app->share(function ($app) use ($handlers) {
-            return new Logger('main', $handlers);
-        });
+        // Syslog Handler
+        $syslogIdent = Arr::path($this->config, 'syslog.ident');
 
-        $app->initializer('Synapse\\Log\\LoggerAwareInterface', function ($object, $app) {
-            $object->setLogger($app['log']);
-            return $object;
-        });
+        if ($syslogIdent) {
+            $handlers[] = $this->syslogHandler($syslogIdent);
+        }
+
+        return $handlers;
     }
 
-    /**
-     * Perform extra chores on boot (none needed here)
-     *
-     * @param  Application $app
-     */
-    public function boot(Application $app)
+    protected function syslogHandler($ident)
```

Comment:
OCD, but function names should be verbs. Can we rename all these to `getFooHandler`?